en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/failover-endpoints.md

```diff
@@ -0,0 +1,117 @@
+# Failover Endpoints.
+
+Failover endpoints provide uninterrupted API invocation capability to the users even though there is a failure in a given API endpoint. When failover is enabled, the traffic that comes to the relevant resource is always routed to the first endpoint. Thereafter, the traffic is routed to second endpoint only if the first endpoint is not available.
+
+Similar to the load balancing, Choreo Connect (CC) provides failover capability with below two WSO2 product combinations.
+
+- [Failover Endpoints With APIM]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/failover-endpoints/#failover-endpoints-with-apim). 
+- [Failover Endpoints With APICTL]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/failover-endpoints/#failover-endpoints-with-apictl).
+
+Below section explains how to configure failover endpoints considering above products.
+
+## Failover Endpoints With APIM
+
+!!! Important
+    Currently WSO2 API Manager allows to add failover endpoints only to the API level.
+
+### Step 1 - Define Failover Endpoints In Publisher
+
+After creating an API in the APIM publisher, select the API you want to provide failover capability.
+
+1.  Click **Develop -> API Configurations -> Endpoints**.
+
+    1.1. Under the **Load balance and Failover Configurations** and select the endpoint type as **Failover**.
```

Comment:
Let's number sub points as 1. 2. 3. This will render as a. b. c. in the docs. Please fix for all sub points in the doc.